packages/ns-api/files/ns.backup

```diff
@@ -187,7 +187,7 @@ elif cmd == 'call':
         except KeyError as error:
             print(json.dumps(utils.validation_error('file', 'required')))
         except RuntimeError as error:
-            print(json.dumps(utils.generic_error(error.args[0])))
+            print(json.dumps(utils.validation_error('passphrase',error.args[0])))
```

Comment:
I think this is not correct: if the backup is corrupted, and error on passphrase is returned.

Reply:
this is what we have already done [L132 ](https://github.com/NethServer/nethsecurity/blob/86d38d08f849b1d8deca9377944baae330ab5414/packages/ns-api/files/ns.backup#L132)

it is the same accepted solution, two error messages possible

invalid backup or empty passphrase (but required)
wrong passphrase

Reply:
it comes from https://github.com/NethServer/nethsecurity/blob/86d38d08f849b1d8deca9377944baae330ab5414/packages/ns-api/files/ns.backup#L70

only two messages are sent back

Reply:
> this is what we have already done L132 

So we should fix also that one.

> it is the same accepted solution, two error messages possible

we can use the messages to raise the correct exception, or change the restore_backup function to raise 2 different exceptions

Reply:
The error is still there: if the restore fails, a validation error is raised:
![image](https://github.com/NethServer/nethsecurity/assets/804596/611ae1b2-98f4-43dc-86cf-caf3ce3e23d8)